CHANGELOG.md

```diff
@@ -126,6 +126,26 @@
 - Optimised creation of bit arrays on the JavaScript target.
   ([Richard Viney](https://github.com/richard-viney))
 
+- The compiler can now infer the variant of custom types when pattern matching,
+  allowing for safe record updates if the variant is known, or access fields
+  which are only present on one variant:
```

Comment:
Instead of cramming it into a single entry I'd split it in two: one for the record updates and one for the record access, so the example can be more focused

Reply:
Good idea, I wasn't super happy with how the entry looked

Reply:
It'd also need to bring up pattern matching!